### ISSUE TITLE
Potentially add a module on 'Using SQL with your own data'

### ISSUE THREAD

**Issue by tracykteal** [datacarpentry/sql-ecology#45]
This module would be the last one and would discuss how to use SQL with your own data. It could

- Emphasize that SQLite is fine for their own work
- Give an overview of what they would do, i.e. import your own CSV files like we did here
- List the potential limitations of SQLite 
- List other database types and some of their strengths/weaknesses (not overwhelmingly technical, more just like "handles databases up to 10GB well", "is good for multiple users", "need access to a Linux server to use and install", "command line interface only")
- Discuss that once a database is created, you can connect to it through other languages like R or Python (and that we'll demonstrate that in the capstone)
- Have some links to materials or info on good database design

**Comment by tpoisot** [comment]
Could some it be done as a capstone on "From Excel to SQL"? I'll have to write that (tentatively) for a course I'm giving.

**Comment by ethanwhite** [comment]
:+1: @tpoisot if your doing this anyway and would like to contribute it I think that would be awesome!

**Comment by tpoisot** [comment]
Right, I'll do, but it might take a while. *I.e.* until I start preparing this class of mine.
